sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -3,7 +3,7 @@
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 """Test grid search CV."""
 
-__author__ = ["mloning", "fkiraly"]
+__author__ = ["mloning", "fkiraly", "hazrul"]
```

Comment:
@hazrulakmal, no?

Reply:
ops sorry about that didn't exactly remember why I put my name there. fixed it

Reply:
no, I meant, it should be @hazrulakmal, not hazrul, because @hazrulakmal is your GitHub user name (not that you should remove your name). It's fine to add yourself as an author if you wrote significant code in the module.

Reply:
I thought you meant authors are for the original author who started the module. my bad for misinterpreting. anyway, It's fine, I can add it later.

Reply:
well, strictly speaking, there is ownership per estimator (by file), the authors line in tests etc is more for acknowledgment.